src/update.ts

```diff
@@ -74,6 +88,13 @@ export class UpdateQuery<
       .catch(onRejected);
   }
 
+  withComment(comment: string, removeSpace?: boolean) {
+    assert(!comment.includes('*'))
```

Comment:
I think we can be more flexible and provide a slightly better error message?

```ts
// Declare at the top level, to avoid having to recompile on each use.
const endCommentRe = /\*\//;

// Then in the function.
const m = endCommentRe.exec(comment);
if (m !== null) throw new Error('Found "*/" in comment contents.');
```